Fundamentals/Backoffice/Property-Editors/Built-in-Property-Editors/Block-Grid-Editor/index.md

```diff
@@ -0,0 +1,670 @@
+- **Layout Stylesheet** - Replace the built-in Layout Stylesheet. Additionally, you can retrieve the default layout stylesheet to use as a base for your own or inspiration for writing your own.
+- **Create Button Label** - Overwrite the label of the create button.
+
+## Setup Block Types
+
+Block Types are based on **Element Types**. These can be created beforehand or while setting up your Block Types.
+
+Once you have added an Element Type as a Block Type on your Block Grid Data Type you have the option to configure it.
+
+![Block List - Data Type Block Configuration](images/BlockListEditor_DataType_Blocks.png)
+> TODO: screenshot
+
+### Groups
+Blocks can also be grouped, this will be visible in the Block Catalogue. This can also be used to allow a group of Blocks in a Area.
+
+## Block Configuration
+Each Block has a set of properties that are optional to configure. These are described below.
+
+### General
+
+Customize the user experience for your content editors when they work with the blocks in the Content section.
+
+- **Label** - Define a label for the appearance of the Block in the editor. The label can use AngularJS template-string-syntax to display values of properties.
+  :::tip
+  Label example: "My Block {{myPropertyAlias}}" will be shown as: "My Block FooBar". 
+  
+  You can also use more advanced expression using AngularJS filters, like `{{myPropertyAlias | limitTo:100}}` or for a property using Richtext editor `{{myPropertyAlias | ncRichText | truncate:true:100}}`. It is also possible to use properties from the Settings model by using `{{$settings.propertyAlias}}`.
+  :::
+
+
+- **Content model** - This presents the Element Type used as model for the Content section of this Block. This cannot be changed, but you can open the Element Type to perform edits or view the properties available. Useful when writing your Label.
+- **Settings model** - Add a Settings section to your Block based on a given Element Type. When selected you can open the Element Type or choose to remove the Settings section again.
+    
+### Size options
+
+Customize the Blocks size in the Grid, if you define multiple options the Block becomes scalable.
+
+- **Available column spans i** - Define one or more columns this Block spans across. Example in a 12 columns grid, 6 columns is equivalent to half width. By enabling 6 columns and 12 columns, the Block can be scaled to either be half width or full width.
+- **Available row spans i** - Define the amount of rows this Block spans across.
+
+
+### Catalogue appearance
+
+These properties refer to how the Block is presented in the Block catalogue when editors choose which Blocks to use for their content.
+
+- **Background color** - Define a background color to be displayed beneath the icon or thumbnail. Example: `#424242`.
+- **Icon color** - Change the color of the Element Type icon. Example: `#242424`.
+- **Thumbnail** - Pick an image or SVG file to replace the icon of this Block in the catalogue.
```

Comment:
Nice. Never knew I could pick a custom SVG thats not registered via the usual way of the default icons or adding a custom one with the folder convention in App_Plugins - so good to see this documented 👍